### PR TITLE
TELCODOCS-887 412 Telco Core RN bug fixes

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-887]: Document 412 fixed issues for Telco core

Version(s):
4.12
Issue: https://issues.redhat.com/browse/TELCODOCS-887

Link to docs preview: https://53941--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-bug-fixes

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -863,8 +870,18 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
...
+
+* Previously, when the Tuned daemon was restarted out of order as part of the cluster Node Tuning Operator, the CPU affinity of interrupt handlers was reset and the tuning was compromised. With this fix, the `irqbalance` plug-in in Tuned is disabled, and {product-title} now relies on the logic and interaction between `CRI-O` and `irqbalance`. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2105123[*BZ#2105123*].
+
+* Previously, the rollout times for `kube-apiserver` on loaded clusters were slow and sometimes exceeded the 5-minute rollout timeout. With this fix, the rollout times should be shorter and within the 5-minute threshold. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2109965[*BZ#2109965*].
```
Comment (MarSik) [inline release_notes/ocp-4-12-release-notes.adoc:882]: This bug is a symtom only of a bigger issue that we fixed. I think we should focus more on the underlying issue in the RN. Please check out the PR description of https://github.com/openshift/cluster-node-tuning-operator/pull/479 - simply said, we were executing a hook script for every new veth device created during a pod startup. And the internal logic took too long when the node was under load. These delays accumulated during pod start events which lead to the mentioned bug (timeout during rollout).

@Tal-or Could you please work with Kevin on the https://bugzilla.redhat.com/show_bug.cgi?id=2109965 RN and describe the cause, effect, fix, outcome as the initial template suggested please?

Comment (kquinn1204) [reply release_notes/ocp-4-12-release-notes.adoc]: @Tal-or Can you review the updated description please.

**Comment by kquinn1204** [comment]
Got LGTM for BZ-2109965 on slack from @Tal-or "The description seems correct and clear to me and I'm OK with it."

**Comment by kquinn1204** [comment]
Got LGTM in slack from @jlojosnegros on OCPBUGS-3475 "The Tuned profile has been modified ..."

**Comment by kquinn1204** [comment]
HI @MarSik I updated the description are you happy with the other descriptions as I see you listed in dialog on tickets. Can you set a LGTM If you are OK with the descriptions.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -625,6 +625,13 @@ For more information, see xref:../networking/network_observability/network-obser
 [id="ocp-4-12-storage"]
 === Storage
 
+[id="ocp-4-12-low-latency-tuning"]
+=== Low latency tuning
+
+[id="ocp-4-12-low-latency-tuning-features-options"]
+==== Support for new kernel features and options
+The low latency tuning has been updated to use the latest kernel features and options. The fix for link:https://bugzilla.redhat.com/show_bug.cgi?id=2117780[2117780] introduced a new per-CPU `kthread`, `ktimers`. This thread must be pinned to the proper CPU cores. With this update, there is no functional; the isolation of the workload is the same. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2102450 [2102450].
```
Comment (tmalove) [inline release_notes/ocp-4-12-release-notes.adoc:633]:
```suggestion
The low latency tuning has been updated to use the latest kernel features and options. The fix for link:https://bugzilla.redhat.com/show_bug.cgi?id=2117780[2117780] introduced a new per-CPU `kthread`, and `ktimers`. This thread must be pinned to the proper CPU cores. With this update, there is no functional. The isolation of the workload is the same. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2102450 [2102450].
```
In the sentence that starts with 'With this update...', there seems to be something missing after 'functional'.

**Review by tmalove — APPROVED** [review]

**Pull request merged**